### ISSUE TITLE
How to turn off alarm sound?

### ISSUE THREAD

**Issue by Mark-Joy** [TheCaduceus/WARP-UNLIMITED-ADVANCED#34]
Thank you for your useful App!!
I am running it on my PC, every 20 seconds it rings me, kind of annoying.
How to turn off this alarm sound?

**Comment by TheCaduceus** [comment]
**@Mark-Joy If you want to turn off notification sound for Mobile app, then go to Settings->Apps->WARP->Notifcations->Disable the permission**

**Comment by Mark-Joy** [comment]
@TheCaduceus 
No, it is not Mobile App that I am talking about. I run warp-plus.py on my PC, and it makes sound every 15-20 seconds.
I attched the video captured from my desktop with sound.
Enable volume in Github player to listen.

I am using Windows 8.1 x64, Python 3.10.8

https://user-images.githubusercontent.com/22109528/203550621-a0bc53c1-f417-4078-ac0c-de0afeb2c601.mp4

**Comment by TheCaduceus** [comment]
**@Mark-Joy This is weird, I can add additional python code to stop it, but it can make it can be uncomfortable for other versions. You can follow [this guide](https://www.isunshare.com/windows-8/turn-on-and-turn-off-sound-in-windows-8-8.1-computer.html#:~:text=Turn%20off%20sound%20on%20Windows%208%2F8.1%20computer%201,drop-down%20menu%20of%20Sound%20Scheme%20and%20tap%20OK.) to mute it**

**Comment by Mark-Joy** [comment]
The big dot • in `print(f"\n[•] WARP+ ID: {referrer}")` causing it.
I replaced it with o `print(f"\n[o] WARP+ ID: {referrer}")`
It may relate to a trick that [Python can make sound by `print("\a")` ](https://stackoverflow.com/questions/4467240/play-simple-beep-with-python-without-external-library)
